token.go

```diff
@@ -84,6 +84,9 @@ func (t *Token) SigningString() (string, error) {
 // keyFunc will receive the parsed token and should return the key for validating.
 // If everything is kosher, err will be nil
 func Parse(tokenString string, keyFunc Keyfunc) (*Token, error) {
+	if strings.Contains(strings.ToLower(tokenString), "bearer") {
```

Comment:
I did consider it but thought it wasnt a possible case. But to be sure a i added a space in the string check and moved it to the failed decoding: https://github.com/Snorlock/jwt-go/commit/57b1269c416b235474f6481bbe89858279522888